tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -16,11 +16,14 @@ class StatusBar(PageRegion):
 
     @property
     def height(self):
-        return self.root_element.rect['height']
+        if self.is_displayed:
+            return self.root_element.rect['height']
+        else:
+            return 0
 
     @property
     def is_displayed(self):
-        return self.root_element.is_displayed()
+            return self.root_element.is_displayed() and self.root_element.rect['y'] == 0
```

Comment:
I thought that when an element is outside of the viewport .is_displayed() returns False. Is that not the case?